apps/backend/src/mcp/tools/device_management.py

```
@@ -26,6 +26,150 @@
+                    "message": f"Device '{hostname}' added successfully to infrastructure registry",
+                    "connectivity_test": {
+                        "performed": monitoring_enabled,
+                        "status": device_info.get("status", "unknown"),
+                        "ssh_accessible": device_info.get("status") == "online"
+                    },
+                    "next_steps": [
+                        "Device is now available for monitoring",
+                        "Use other MCP tools to manage containers, check system metrics, etc.",
+                        "Update device configuration via update_device tool if needed"
+                    ],
+                    "timestamp": datetime.now(timezone.utc).isoformat()
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)